lib/elix/message_scheduler.ex

```diff
@@ -64,4 +71,14 @@ defmodule Elix.MessageScheduler do
   defp heartbeat do
     Process.send_after(self(), :heartbeat, :timer.seconds(1))
   end
+
+  # This is unnecessary for the process brain. I feel like
```

Comment:
Oh yeah, and I can totally do this now, as the Redis brain [already handles encoding and decoding](https://github.com/stevegrossi/hedwig_brain/blob/master/lib/brain/redis_store.ex#L64) itself. Good catch!